packages/labs/react/package.json

```diff
@@ -30,18 +30,113 @@
     "/use-controller.{d.ts,d.ts.map,js,js.map}"
   ],
   "scripts": {
-    "build": "npm run clean && npm run build:ts --build && rollup -c",
-    "build:watch": "rollup -c --watch",
-    "build:ts": "tsc --build && treemirror development . '**/*.d.ts{,.map}'",
-    "build:ts:watch": "tsc --build --watch",
-    "clean": "rm -rf {index,create-component,use-controller}.{js,js.map,d.ts} development/ test/ *.tsbuildinfo",
-    "dev": "scripts/dev.sh",
-    "test": "npm run test:dev && npm run test:prod",
-    "test:dev": "MODE=dev node ../../tests/run-web-tests.js \"development/**/*_test.js\" --config ../../tests/web-test-runner.config.js",
-    "test:prod": "MODE=prod node ../../tests/run-web-tests.js \"development/**/*_test.js\" --config ../../tests/web-test-runner.config.js",
-    "test:watch": "npm run test:dev -- --watch",
-    "checksize": "rollup -c --environment=CHECKSIZE",
-    "regen-package-lock": "rm -rf node_modules package-lock.json; npm install"
+    "build": "wireit",
+    "build:ts": "wireit",
+    "build:ts:types": "wireit",
+    "build:rollup": "wireit",
+    "test": "wireit",
+    "test:dev": "wireit",
+    "test:prod": "wireit",
+    "checksize": "wireit"
+  },
+  "wireit": {
+    "build": {
+      "dependencies": [
+        "build:rollup",
+        "build:ts",
+        "build:ts:types"
+      ]
+    },
+    "build:ts": {
+      "command": "tsc --build --pretty",
+      "dependencies": [
+        "../../reactive-element:build:ts:types"
+      ],
+      "clean": "if-file-deleted",
+      "files": [
+        "src/**/*.ts",
+        "src/**/*.tsx",
+        "tsconfig.json"
+      ],
+      "output": [
+        "development",
+        "tsconfig.tsbuildinfo"
+      ]
+    },
+    "build:ts:types": {
+      "command": "treemirror development . \"**/*.d.ts{,.map}\"",
+      "dependencies": [
+        "../../internal-scripts:build",
+        "build:ts"
+      ],
+      "files": [],
+      "output": [
+        "*.d.ts{,.map}",
+        "test/**/*.d.ts{,.map}"
+      ]
+    },
+    "build:rollup": {
+      "command": "rollup -c",
+      "dependencies": [
+        "build:ts",
+        "../../reactive-element:build:rollup"
+      ],
+      "files": [
+        "rollup.config.js",
+        "../../rollup-common.js"
```

Comment:
This is in labs and needs an extra relative `../`. This looks for a file in `/packages/rollup-common.js` that doesn't exist.

I.e. This should be: `"../../../rollup-common.js"`